### PR TITLE
Adding a fault wrapper to catch and wrap exceptions that are thrown as p...

### PR DESCRIPTION
**Pull request by rnirmal**
...lain text.

### CODE REVIEW

**Comment by TimSimpsonR** [comment]
+1

**Comment by ed-** [comment]
+1

File: reddwarf/extensions/mysql/service.py [hunk 1]
```diff
@@ -131,11 +131,17 @@ def delete(self, req, tenant_id, instance_id, id):
...
+        try:
+            user = guest_models.MySQLUser()
+            user.name = id
+            models.User.delete(context, instance_id, user.serialize())
+        except ValueError as ve:
+            raise exception.BadRequest(ve.message)
         return wsgi.Result(None, 202)
 
+    def show(self, req, tenant_id, instance_id, id):
+        raise webob.exc.HTTPNotImplemented()
```
Comment (cp16net) [inline reddwarf/extensions/mysql/service.py:143]: is this expected behavior?

Comment (rnirmal) [inline reddwarf/extensions/mysql/service.py:143]: yes

**Comment by rnirmal** [comment]
I should have added more description early on. This is to wrap any exceptions that get thrown back as plain text. Some of it is in the openstack common wsgi stuff and some in auth. I couldn't find a better way than checking for the content-types, otherwise we wrap our valid error messages with default ones. This makes sure that only the ones returned in plain text is wrapped with the default error messages.

**Comment by cp16net** [comment]
+1

**Pull request merged**